src/jit/gentree.h

```diff
@@ -3264,42 +3264,48 @@ struct GenTreeCall final : public GenTree
     }
 
 #define GTF_CALL_M_EXPLICIT_TAILCALL                                                                                   \
-    0x0001                         // GT_CALL -- the call is "tail" prefixed and importer has performed tail call checks
-#define GTF_CALL_M_TAILCALL 0x0002 // GT_CALL -- the call is a tailcall
-#define GTF_CALL_M_VARARGS 0x0004  // GT_CALL -- the call uses varargs ABI
-#define GTF_CALL_M_RETBUFFARG 0x0008        // GT_CALL -- first parameter is the return buffer argument
-#define GTF_CALL_M_DELEGATE_INV 0x0010      // GT_CALL -- call to Delegate.Invoke
-#define GTF_CALL_M_NOGCCHECK 0x0020         // GT_CALL -- not a call for computing full interruptability
-#define GTF_CALL_M_SPECIAL_INTRINSIC 0x0040 // GT_CALL -- function that could be optimized as an intrinsic
-                                            // in special cases. Used to optimize fast way out in morphing
+    0x00000001 // GT_CALL -- the call is "tail" prefixed and importer has performed tail call checks
```

Comment:
> 0x00000001 // GT_CALL -- the call is "tail" prefixed and importer has performed tail call checks [](start = 3, length = 97)

Since you are editing this, see if you can fix the alignment of constants and comment.